media-libs/libavif/libavif-9999.ebuild

```diff
@@ -36,15 +41,21 @@ BDEPEND="
 	virtual/pkgconfig
 "
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.1.0-tests-cxx.patch
```

Comment:
Is this patch still needed in live ebuild after
https://github.com/AOMediaCodec/libavif/commit/3640a3ae3dd0082301c43b3d4d182714a06662d8
?

Reply:
Hello @novomesk,
No, it's not needed anymore.
I've dropped it in 3af4480.